submissions/the-darkesttt/js-post-oop/index.js

```diff
@@ -0,0 +1,92 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/the-darkesttt/a-tiny-JS-world
+   Web app: https://the-darkesttt.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+
+class Creature {
+
+   constructor( name, gender, saying, species) {
+      this.name = name;
+      this.gender = gender;
+      this.saying = saying;
+      this.species = species;
+   }
+
+   showValues() {
+      const valueKeys = [
+         'species',
+         'name',
+         'gender',
+         'saying'
+      ];
+      const inhabitantValues = valueKeys.map((key) => {
+         return `${this[key]}`;
+      });
+      return inhabitantValues;
```

Comment:
`keys` is a technical term. Inhabitants are characterized by rather properties or features. No one says 'Name is my key'.

Also you can chain up all these statements in a single `return` statement.

If an arrow function has a single argument parentheses are not required.
Arrow functions that have a single return statement do not need curly braces and `return` keyword.
Compare:
```
names.map((name) => { return name.toUpperCase(); })

names.map(name => name.toUpperCase())
```

Why `this.key` is wrapped into the template string?